frontend/rust-lib/flowy-test/src/document/document_event.rs

```diff
@@ -138,6 +163,18 @@ impl DocumentEventTest {
       .parse::<DocumentRedoUndoResponsePB>()
   }
 
+  pub async fn apply_delta_for_block(&self, document_id: &str, block_id: &str, delta: String) {
+    let block = self.get_block(document_id, block_id).await;
+    let text_id = block.unwrap().external_id.unwrap();
```

Comment:
Is it safe to force unwrap here?

Reply:
This is a test. I thought if I can't get this test, it will fail directly.